### PR TITLE
Add technical support contact email to applicant footer

### PR DESCRIPTION
**Pull request by bion**
image

### Issue(s)
addresses #1308

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java [hunk 1]
```diff
@@ -228,8 +254,12 @@ protected ContainerTag renderProgramApplicationTitleAndProgressIndicator(
...
    */
   private int getPercentComplete(int blockIndex, int totalBlockCount, boolean forSummary) {
-    if (totalBlockCount == 0) return 100;
-    if (blockIndex == -1) return 0;
+    if (totalBlockCount == 0) {
+      return 100;
+    }
+    if (blockIndex == -1) {
+      return 0;
+    }
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:262]: this was the autoformatter

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java [hunk 2]
```diff
@@ -33,31 +34,56 @@
...
     bundle.addBodyStyles(ApplicantStyles.BODY);
     String currentTitle = bundle.getTitle();
+
     if (currentTitle != null && !currentTitle.isEmpty()) {
       bundle.setTitle(String.format("%s — %s", currentTitle, CIVIFORM_TITLE));
     } else {
       bundle.setTitle(CIVIFORM_TITLE);
     }
+
+    bundle.addFooterStyles(Styles.MT_24);
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:85]: idk if this is the best way to do this, but putting it all the way on the bottom looked easy to miss at least on my big monitor

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java [hunk 3]
```diff
@@ -33,31 +34,56 @@
...
       ViewUtils viewUtils,
       Config configuration,
       ProfileUtils profileUtils,
-      LanguageSelector languageSelector) {
+      LanguageSelector languageSelector,
+      Config appConfig) {
     super(viewUtils, configuration);
     this.profileUtils = checkNotNull(profileUtils);
     this.languageSelector = checkNotNull(languageSelector);
+    this.supportEmail = checkNotNull(appConfig).getString("support_email_address");
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:54]: Question (blocking): How is appConfig different from configuration?

**Review by hmhsu — APPROVED** [review]

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java (hunk 2 above)
Comment (natsid) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:85]: Does giving it a top margin make it look higher up?

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java (hunk 3 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:54]: the `Config` class is somewhat generic. labeling this param as `appConfig` was intended to make it very clear that this is the app-wide config object

**Review by natsid — APPROVED** [review]

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java (hunk 2 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:85]: No it just pushes it away from the main content a bit

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java (hunk 3 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:54]: I meant there is another argument `configuration` in the constructor and I thought the two are the same.

Comment (bion) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java:54]: oof, that's embarrassing. i'll fix it in another PR

**Pull request merged**